latest-git.sh

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+# This is a script used in travis-ci to have latest git
+set -ex
+git clone git://github.com/git/git --depth 1 /tmp/git
+pushd /tmp/git
+make -j 8
+popd
```

Comment:
This (along with the .travis.yml stuf) is lifted directly from https://github.com/pre-commit/pre-commit/blob/60cc4158ed4b4169693b99f70283b18cb25eea63/latest-git.sh